package/MDAnalysis/core/universe.py

```diff
@@ -238,6 +238,18 @@ class Universe(object):
     vdwradii: dict, ``None``, default ``None``
         For use with *guess_bonds*. Supply a dict giving a vdwradii for each
         atom type which are used in guessing bonds.
+    context: string or Guesser, default ``default``
+        Type of the Guesser to be used in attributes guessing
+    to_guess: list, (optional), defualt ``['types', 'masses']``
+              (in future versions types and masses will be removed)
+        Attributes to be guessed (these attributes will be either guessed
+        if they don't exist in the universe or partially guessed by only filling
+        its empty values if universe has the attribute)
+    force_guess: list, (optional)
+        Attributes to be forced guessed (these attributes will be either
+        guessed if they don't exist in the universe or their values will be
+        completely overwritten by guessed ones if the universe has
+        the attribute)
```

Comment:
```suggestion
        TopologyAttrs in this list will be force guessed. If the TopologyAttr
        does not already exist in the Universe, this has no effect. If the TopologyAttr
        does already exist, all values will be overwritten by guessed values.
```

Just suggesting to clarify types here as well!